python/amici/petab_objective.py

```diff
@@ -0,0 +1,236 @@
+"""Functionality related to evaluating the objective function for a PEtab
+problem"""
+
+import numbers
+import numpy as np
+import pandas as pd
+import copy
+
+from typing import List, Sequence
+import amici
+import petab
+
+
+def edatas_from_petab(
```

Comment:
I guess this is just a copy-paste from pypesto?

Reply:
With potentially some cleanup and renaming, yes.

Reply:
ok. i did not check in detail given that the functions used to work beforehand.